spec/fabricators/video_fabricator.rb

```diff
@@ -1,4 +1,8 @@
 Fabricator(:video) do
   title { Faker::Lorem.characters(5) }
   description { Faker::Lorem.sentence }
-end
\ No newline at end of file
+end
+
+Fabricator(:video_upload, from: :video) do
+  large_cover { Rack::Test::UploadedFile.new("./spec/support/uploads/monk_large.jpg") }
```

Comment:
I think Marcin Drozd had nice solution to this in terms of `File.open(...)` - but I am not 100% sure this could work with all drivers. I haven't done this myself, tho.